dipy/align/tests/test_parzenhist.py

```diff
@@ -369,9 +369,10 @@ def test_joint_pdf_gradients_dense():
         mgrad, inside = vf.gradient(moving.astype(np.float32), moving_g2w,
                                     spacing, shape, grid_to_space)
         id = transform.get_identity_parameters()
-        parzen_hist.update_gradient_dense(id, transform, static.astype(np.float64),
-                                     moved.astype(np.float64), grid_to_space,
-                                     mgrad, smask, mmask)
+        parzen_hist.update_gradient_dense(
+            id, transform, static.astype(
```

Comment:
I think in general it is more readable when each parameter is contained within one single line (when possible). For example, it is not necessary to break `static.astype(np.float64)` nor `moved.astype(np.float64)`, what about:
```
        parzen_hist.update_gradient_dense(
            id, transform, static.astype(np.float64),
            moved.astype(np.float64), grid_to_space,
            mgrad, smask, mmask)
```